src/main/java/com/razorpay/ApiClient.java

```diff
@@ -38,6 +38,11 @@
     return processResponse(response);
   }
 
+  <T extends Entity> T put(String path, JSONObject requestObject) throws RazorpayException {
```

Comment:
Need to add formatter for [eclipse](https://github.com/google/styleguide/blob/gh-pages/eclipse-java-google-style.xml)/[intelliJ](https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml). 
Also update the README